### PR TITLE
added dep to project

### LINKED ISSUE #16: Package management setup (dep)

### PR DESCRIPTION
**Pull request by andrewnguyen22**
closes #16

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Gopkg.toml example
```
Comment (luyzdeleon) [inline Gopkg.toml]: Remove example comments.

**Review by luyzdeleon — CHANGES_REQUESTED** [review]
Add vendor/ to gitignore, as every environment needs to build their own based on package management.

File: Gopkg.toml (hunk 1 above)
Comment (andrewnguyen22) [reply Gopkg.toml]: Check it out now

Comment (andrewnguyen22) [reply Gopkg.toml]: Removed .idea

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**